Ladybird/RequestServer/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ target_link_libraries(RequestServer PRIVATE requestserverservice)
 
 target_include_directories(requestserverservice PRIVATE ${LADYBIRD_SOURCE_DIR}/Userland/Services/)
 target_include_directories(requestserverservice PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/..)
-target_link_libraries(requestserverservice PUBLIC LibCore LibMain LibCrypto LibFileSystem LibIPC LibMain LibTLS LibWebView LibWebSocket LibURL LibThreading CURL::libcurl)
+target_link_libraries(requestserverservice PUBLIC LibCore LibMain LibCrypto LibFileSystem LibIPC LibMain LibTLS LibWeb LibWebView LibWebSocket LibURL LibThreading CURL::libcurl)
```

Comment:
> LibWeb LibWebView

Not exactly your fault but...why does RS need to link against anything libweb?

Reply:
This really goes _way_ back: https://github.com/LadybirdBrowser/ladybird/commit/58e3b8cf58edd2f0c68ebdf5617731678d19ea82 (though it makes sense in a weird way)
Also cc @trflynn89...that seems like a hack at best? why not depend directly on `generate_RequestClientEndpoint.h` and `generate_RequestServerEndpoint.h`?

Reply:
We're still doing that (roughly, those are listed in the GENERATED_SOURCES for WebView), but even then there's no real reason to actually link that thing in afaict